clients/include/testing_matmul.hpp

```diff
@@ -900,6 +893,46 @@ hipDataType derive_unset_bias_type(const Arguments& arg)
     return real_bias_type;
 }
 
+// A function to determine the default compute_input_type
+std::tuple<hipDataType, hipDataType> derive_unset_compute_input_type(const Arguments& arg)
+{
+    static const std::set<hipDataType> supported_compute_input_types = {HIP_R_32F,
```

Comment:
Did you miss 64F and 32I?

Reply:
https://github.com/ROCm/hipBLASLt/blob/576d0c76701c32c74686f0a2394d42bb2c6ef469/clients/benchmarks/client.cpp#L391-L398

Even if the user inputs a `compute_input_type[AB]` that is larger than `compute_input`, there is no if-else condition to handle 64F and 32I in the original `type_dispatch.h` logic. It will default to the original tc type.

https://github.com/ROCm/hipBLASLt/blob/3ce620fa674fa8c3eab3af01fe9a2fe7bb5d0f98/clients/include/type_dispatch.hpp#L149

As I understand it, this feature seems to only allow the user to set the `compute_input_type` to be smaller than the compute type. Allowing the user to input the largest type for both float and int seems pointless.